homeassistant/components/camera/__init__.py

```diff
@@ -345,21 +398,54 @@ def state(self):
             return STATE_STREAMING
         return STATE_IDLE
 
+    @property
+    def is_on(self):
+        """Return true if on."""
+        return True
+
+    def turn_off(self):
+        """Turn off camera."""
+        raise NotImplementedError()
+
+    async def async_turn_off(self):
+        """Turn off camera.
+
+        This method must be run in the event loop and returns a coroutine.
+        """
+        return await self.hass.async_add_job(self.turn_off)
```

Comment:
We shouldn't return anything here since we already await the task. The docstring should be updated too to reflect this.

We can now instead use the specific method `hass.async_add_executor_job` since we know that `turn_off` should be scheduled on the thread pool.

Reply:
I have to rebase to get `async_add_executor_job`